### DISCUSSION TITLE
[v3] mb_strtoupper(): Argument #1 ($string) must be of type string, array given

### DISCUSSION THREAD

**Discussion by makspage** [livewire/livewire#6089] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.1

### Steps To Reproduce

If there are several conditions in the rule and validation fails, then this error is thrown: 
mb_strtoupper(): Argument #1 ($string) must be of type string, array given 
```php
<?php

namespace App\Livewire;

use Livewire\Attributes\Rule;
use Livewire\Form;

class TestForm extends Form
{
    #[Rule('required|numeric|min:1')]
    public $amount = '';

    public function store()
    {
        $this->validate();
        //...
    }

}
```
livewire version v3.0.0 beta.5

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@makspage thanks for reporting! Any chance you could submit a failing test PR so we can investigate? Thanks!
